Modules/Module_Import/Import/xASL_imp_DetermineSubjectStructure.m

```diff
@@ -153,10 +148,19 @@
         warning('x.SUBJECTS is undefined, data loading is going to fail...');
     end
     
-
+	% If tokenVisitAliases is not provided, we assume visits are not renamed and create a dummy tokenVisitAliases list
+	if ~isfield(x.modules.import.imPar, 'tokenVisitAliases') || isempty(x.modules.import.imPar.tokenVisitAliases)
+		% Create a unique list of sessions
+		listSessions = unique(listSessions);
+
+		% Exclude empty session names
+		listSessions = listSessions(~ismember(listSessions,''));
```

Comment:
What will happen to sub-001/ses-<'illegalsymbols'> ? The symbols will be removed above and this sub-visit will crash somewhere down the line as it's session cannot be found?

Reply:
Actually, it shouldn't crash. This here creates a dummy tokenVisitAlias list for the case we don't rename. Session with all illegal characters will be skipped from this list here. So later on, we will not be able to identify them and it will just skip them. 

We can try to create a warning for this: In case that we go for non-renaming Visits, we will specifically check for sessions that turn out empty after removing special characters and we will report this to the user. 

Is that fine? To me, a session without any alphanumerical characters is so exotic I would just ignore it, but I'm happy to add this warning if you think this might occur.

Reply:
Yes if it's not too much work. Can we integrate this with my above comment with empty BIDS session values?

(in BIDS key = session/visit, value = the visit name)

Reply:
we now name empty visits as missingSessionValue.

All user output uses BIDS terminology.